src/bundle/Resources/public/js/scripts/button.state.checkbox.toggle.js

```diff
@@ -0,0 +1,27 @@
+(function (global, doc) {
+    const toggleForms = doc.querySelectorAll('.ez-toggle-btn-state-checkbox');
+    const ALL_CHECKED = 'all-checked';
+    const ANY_CHECKED = 'any-checked';
+
+    const toggleButtonState = (toggleForm, checkboxInputs) => {
+        const button = doc.querySelector(toggleForm.dataset.toggleButtonId);
+        const anyIsChecked = checkboxInputs.some(el => el.checked);
+        const allChecked = checkboxInputs.every(el => el.checked);
+        const toggleMode = toggleForm.dataset.toggleMode || ANY_CHECKED;
+        let methodName = 'setAttribute';
+
+        if ((ALL_CHECKED === toggleMode && allChecked) || (ANY_CHECKED === toggleMode && anyIsChecked)) {
+            methodName = 'removeAttribute';
+        }
+
+        button[methodName]('disabled', true);
+    };
+
+    toggleForms.forEach((toggleForm) => {
+        const checkboxInputs = toggleForm.querySelectorAll('input[type="checkbox"]');
+
+        checkboxInputs.forEach((input, index, inputs) => {
+            input.addEventListener('change', toggleButtonState.bind(null, toggleForm, [...inputs]), false);
```

Comment:
Do you need `[...inputs]`?

Reply:
I can't use `.some` and `.every` on Node Collection, unless there is some other way to do this.